file_paths.md

```diff
@@ -0,0 +1,108 @@
+Puppet Installation Layout
+==========================
+
+This table specifies the file paths in an installation of Puppet and the corresponding Puppet setting, denoted as `:setting`. The 3.x column shows the difference of the specification vs the actual 3.x implementation.
+
+puppet-agent
+------------
+
+    Path                                  Setting                        3.x
+    /etc/puppetlabs                       # :puppetdir                   n/a
```

Comment:
Yeah, I was in favor of `puppetdir` before we decided to use 'puppet' as the name of the directory that `confdir` points to, but I agree that it seems like it would be confusing now, given the latest state of all the paths.